packages/build-info/src/frameworks/tanstack-router.ts

```diff
@@ -0,0 +1,26 @@
+import { BaseFramework, Category, Framework } from './framework.js'
+
+export class TanStackRouter extends BaseFramework implements Framework {
+  readonly id = 'tanstack-router'
+  name = 'TanStack Router'
+  npmDependencies = ['@tanstack/react-router']
+  excludedNpmDependencies = ['@tanstack/start']
+  category = Category.SSG
+
+  dev = {
+    command: 'vite',
```

Comment:
See comment on other PR
```suggestion
    command: 'vite dev',
```